tests/test_gail.py

```diff
@@ -102,6 +107,16 @@ def test_pretrain_images():
     del dataset, model, env
 
 
+def test_gail_callback():
+    dataset = ExpertDataset(expert_path=EXPERT_PATH_PENDULUM, traj_limitation=10,
+                            sequential_preprocessing=True, verbose=0)
+    model = GAIL("MlpPolicy", "Pendulum-v0", dataset)
+    checkpoint_callback = CheckpointCallback(save_freq=500, save_path='./logs/gail/', name_prefix='gail')
```

Comment:
Nit: maybe use a [TemporaryDirectory](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) instance rather than a hardcoded path? This would remove the need to `rmtree` afterwards as the contextmanager automatically cleans it up.

We use hardcoded paths all over the tests, but one day I dream of changing this -- it's caused me headaches in the past from race conditions between tests when trying to parallelize.